### PR TITLE
Clippy suggestions

### PR DESCRIPTION
**Pull request by snim2**
Minor improvements suggested by Clippy. I'm never really sure whether it's sensible to keep these commits separate (for merging, clearly it's better to separate them for reviewing), but potentially they could be collapsed into a single commit.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -361,6 +349,27 @@ fn main() {
             exit_with_message(&format!("Requested parser {:?} does not exist.", parser2));
         }
     }
+    (lexer1, parser1, lexer2, parser2)
+}
+
+fn main() {
+    let argv: Vec<_> = env::args().collect();
+    let args: Args = Docopt::new(USAGE)
+        .and_then(|d| d.argv(argv).decode())
```
Comment (ltratt) [inline src/main.rs]: Not indented properly?

**Comment by ltratt** [comment]
Is this ready for squashing? [I think so, but am not sure...]

**Comment by snim2** [comment]
If you are happy then yes. Do you think it is sensible to squash a PR like this into a single commit (I'm never sure)?

**Comment by ltratt** [comment]
I don't think there's any one-size-fits-all rule for this sort of squashing. I like to see logically distinct commits with messages that explain clearly what's going on; sometimes there's only one such commit; sometimes there are several. So up to you what you think best in this case.

**Comment by snim2** [comment]
OK, squashed!

**Pull request merged**